src/DataSet/DataSet.php

```diff
@@ -70,6 +90,15 @@ public function deleteIndices() {
 	public function buildIndices() {
 		$verify = [];
 		foreach ($this->fixture as $index => $types) {
+
+			if (!$this->connection->getConnection()->indices()->exists(compact('index'))) {
```

Comment:
@dmatsingerzumba the tearDown should be deleting the indexes. I mean this is a good check to do, but not sure how much over head this will add.